README.md

```diff
@@ -45,3 +45,123 @@ Once the above variables are set and the /payload directory has been populated,
+See [the build instructions](#build) for more information on building the image.
+
+You can then run the following command in the root of this git repository
+```shell script
+operator-sdk bundle create $BUNDLE_REPOSITORY:$VERSION_TAG -d deploy/olm-catalog/windows-machine-config-operator/0.0.0 \
+--channels alpha --default-channel alpha --image-builder podman
+```
+The variables in the command should be changed to match the container image repository you wish to store the bundle in.
+You can also change the channels based on the release status of the operator.
+
+You should then push the image to the remote repository
+```shell script
+podman push $BUNDLE_REPOSITORY:$BUNDLE_VERSION
+```
+
+You should verify that the new bundle is valid:
+```shell script
+operator-sdk bundle validate $BUNDLE_REPOSITORY:$BUNDLE_VERSION --image-builder podman
+```
+
+### Creating a new operator index
+An operator index is a collection of bundles. Creating one is required if you wish to deploy your operator on your own
+cluster.
+
+```shell script
+opm index add --bundles $BUNDLE_REPOSITORY:$VERSION_TAG --tag $INDEX_VERSION
+```
+
+#### Editing an existing operator index
+An existing operator index can have bundles added to it:
+```shell script
+opm index add --from-index $INDEX_VERSION
+```
+and removed from it:
+```shell script
+opm index rm --from-index $INDEX_VERSION
+```
+
+### Deploying the operator on a local cluster
+#### Openshift Console
+This deployment method is currently not supported. Please use the [CLI](#cli)
+
+#### CLI
+Change `deploy/olm-catalog/catalogsource.yaml` to point to the operator index created above. Now deploy it:
+```shell script
+oc apply -f deploy/olm-catalog/catalogsource.yaml
+```
+
+This will deploy a CatalogSource object in the `openshift-marketplace` namespace. You can check the status of it via:
+```shell script
+oc describe catalogsource wmco -n openshift-marketplace
+```
+
+Now wait 1-10 minutes for the catalogsource's `status.connectionState.lastObservedState` field to be set to READY.
+
+Create the windows-machine-config-operator namespace:
+```shell script
+oc apply -f deploy/namespace.yaml
+```
+
+Switch to the windows-machine-config-operator project:
+```shell script
+oc project windows-machine-config-operator
+```
+
+Create the OperatorGroup for the namespace:
+```shell script
+oc apply -f deploy/olm-catalog/operatorgroup.yaml
+```
+
+Create the cloud provider and cloud private key secrets. The cloud-private-key should match the keypair used in the
+Windows Machine Config CR you will use.
+```shell script
+# Change paths as necessary
+oc create secret generic cloud-credentials --from-file=credentials=/home/$user/.aws/credentials
+oc create secret generic cloud-private-key --from-file=private-key.pem=/home/$user/.ssh/$keyname
+```
+
+Put the kubeconfig you wish to use in a secret. In order to use the permissions of the windows-machine-config-operator
```

Comment:
So the service account will be generated at a later [step](https://github.com/openshift/windows-machine-config-operator/pull/17/files#diff-04c6e90faac2675aa89e2176d2eec7d8R166) as part of csv creation whereas we want to generate the secret here with sa permissions here?

Reply:
Yes the service account is defined in the `permissions` section of the csv which is generated